pkg/minikube/constants/constants.go

```diff
@@ -35,6 +35,7 @@ const (
 	DefaultCPUS                  = 2
 	DefaultDiskSize              = "20g"
 	UpdateMarkerFileName         = "updated"
+	DefaultISO                   = ""
```

Comment:
This does not look right as B2D is the default ISO. I understand that the default thing is handled in start.go  but the question is if any other code uses this if yes, then we should fix it accordingly and see if we need this at all.